rotkehlchen/inquirer.py

```diff
@@ -920,6 +927,68 @@ def find_curve_pool_price(
 
         return (total_assets_value / total_supply) * (10 ** lp_token.get_decimals())
 
+    def find_gearbox_price(self, token: EvmToken) -> Price | None:
+        evm_manager = self.get_evm_manager(chain_id=token.chain_id)
+        node_inquirer = evm_manager.node_inquirer
```

Comment:
do not have variables used in only one place
```suggestion
        node_inquirer = self.get_evm_manager(chain_id=token.chain_id).node_inquirer
```